cmd/security-agent/common/metrics.go

```diff
@@ -0,0 +1,27 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
This is only used by cmd/security-agent/app so we can probably simply create a file `metrics.go` in `cmd/security-agent/app` and not export `SendRunningMetrics`

Reply:
Done.